phoenix-postgres-pgadmin/example_app/Dockerfile

```diff
@@ -0,0 +1,65 @@
+
+# ---- Build Stage ----
+FROM elixir:1.9-alpine AS app_builder
+
+# Set environment variables for building the application
+ENV MIX_ENV=prod
+ENV LANG=C.UTF-8
+
+# install build dependencies
+RUN apk add --update git build-base nodejs npm yarn python
+
+# Install hex and rebar
+RUN mix local.hex --force && \
+    mix local.rebar --force
+
+# Create the application build directory
+RUN mkdir /app
+WORKDIR /app
+
+# install mix dependencies
+COPY mix.exs mix.lock ./
+COPY config config
+RUN mix deps.get
+RUN mix release.init
+RUN mix deps.compile --force --only $MIX_ENV 
+
+# build assets
+COPY assets assets
+COPY priv priv
+RUN cd assets && npm install && npm run deploy
+RUN mix phx.digest
+
+# build project
+COPY lib lib
+RUN mix compile
+
+# build release
+RUN mix release
```

Comment:
Maybe we can group the `COPY` commands together starting with the one which changes the less to benefit of the layering cache?
Something like that, if that make sens of course
```dockerfile
# build assets
COPY priv priv
COPY lib lib
COPY assets assets
RUN cd assets && npm install && npm run deploy
RUN mix phx.digest \
    && mix compile \
    && mix release
```

Reply:
I grouped the commands that makes sense to executed together in order to take advantage of the caching memory